client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js

```diff
@@ -51,15 +51,32 @@ function LaptopForm() {
 
 		return true;
 	}
+	const closeModale = document.querySelector(".close_modale");
+	const modale = document.querySelector(".modale-content");
+	const form = document.querySelector(".form");
+	//function display greeting after submit the from
+
 	function messageAlert() {
-		if (validateForm(document.laptopRequestForm)) {
-			alert(
-				"thank you for your completing the request form. your request have been recived "
-			);
-		}
+		console.log("heloo");
+		document.body.style.backgroundColor = "black";
+		modale.style.display = "block";
+		form.style.display = "none";
+	}
+	function messageClosed() {
+		console.log("heloo");
+		modale.style.display = "none";
 	}
 	return (
 		<div className="form-card">
+			<div className="modale-content">
```

Comment:
instead add the display block/none in here, based on a useState property that determines whether the form is visible or not

Reply:
do you mean i can make display block/none property with useState.

Reply:
thank you for your feedback